src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/helpers.js

```diff
@@ -78,6 +78,7 @@ function displaySelectedMethod(type) {
     'googlepay',
     'amazonpay',
     'applepay',
+    'cashapp',
```

Comment:
can we move this array to some constants file?

Reply:
Yeah, that's a good idea!